### PR TITLE
🐛 Separate auto-migration e2e from the import tests

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
**What this PR does / why we need it**:

Due to migration suite running together with import tests, the chart state in the same bootstrap cluster was modified from one test to another. This was causing e2e flakes, and this change should fix it.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**:

- [ ] squashed commits into logical changes
- [ ] includes documentation
- [ ] adds unit tests
- [x] adds or updates e2e tests

### CODE REVIEW

File: test/e2e/suites/import-gitops/import_gitops_test.go [hunk 1]
```diff
@@ -32,41 +32,6 @@ import (
 	"github.com/rancher/turtles/test/e2e/specs"
 )
 
-var _ = Describe("[Docker] [Kubeadm] - [management.cattle.io/v3] Migrate v1 to management v3 cluster functionality should work", Label(e2e.ShortTestLabel), func() {
-	BeforeEach(func() {
-		komega.SetClient(setupClusterResult.BootstrapClusterProxy.GetClient())
```
Comment (furkatgofurov7) [inline test/e2e/suites/import-gitops/import_gitops_test.go:37]: can you remove komega package import from the import list if not needed?

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: test/e2e/suites/import-gitops/import_gitops_test.go (hunk 1 above)
Comment (Danil-Grigorev) [reply test/e2e/suites/import-gitops/import_gitops_test.go:37]: Yep, that caused a failure. Will push and try again.

Comment (Danil-Grigorev) [reply test/e2e/suites/import-gitops/import_gitops_test.go:37]: Should be passing now 🙏🏻

**Comment by furkatgofurov7** [comment]
@Danil-Grigorev https://github.com/rancher/turtles/pull/582 is not passing with these changes on top?

**Review by furkatgofurov7 — APPROVED** [review]

**Comment by furkatgofurov7** [comment]
I am fine merging this (since it fixes the current e2e test failure) and building on top of it the other changes

**Review by salasberryfin — APPROVED** [review]

**Pull request merged**